.clang-format

```diff
@@ -0,0 +1,156 @@
+---
+Language:        Cpp
+# BasedOnStyle:  Google
+AccessModifierOffset: -1
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: false
+AlignConsecutiveDeclarations: false
```

Comment:
AlignConsecutiveMacros: true is not here. Without it, our macro defs will look pretty different, just a call-out.

Reply:
That flag isn't available in version 9, were you using 11?

Reply:
I see it on the style page for 9? https://releases.llvm.org/9.0.0/tools/clang/docs/ClangFormatStyleOptions.html

Reply:
Strange, I must have been running on 6, added `AlignConsecutiveMacros: true` back in.